spring-integration-file/src/main/java/org/springframework/integration/file/filters/AbstractPersistentAcceptOnceFileListFilter.java

```diff
@@ -173,7 +173,7 @@ protected void flushIfNeeded() {
 			try {
 				this.flushableStore.flush();
 			}
-			catch (IOException e) {
+			catch (@SuppressWarnings("unused") IOException e) {
 				// store's responsibility to log
```

Comment:
Can't we just re-throw as an `UncheckedIOException` ?

Reply:
I don't think we want to throw an exception here.